data/human/news.txt

```diff
@@ -2190,3 +2190,342 @@ news "graffiti on spaceport wall"
+			"chanting a list of their demands, including ${Worker's Demands}. "
+			"sizing up the planetary security forces. "
+		word
+			"Security eventually steps in, "
+		word
+			"but they're pushed back. "
+			"but they're routed. "
+			"and they disperse the demonstrators. "
+			"and the demonstrators scatter but don't appear dissuaded."
+		replace
+			"%The rioters" "The protesters"
+			"%" ""
+
+news "Clandestine"
+	location
+		planet "Alvorada"
+	name
+		word
+			"Observation"
+	message
+		word
+			"You can't escape the feeling of being watched. It"
+		word
+			"'s probably the security cameras; most of them are still running."
+			" feels like there are eyes around every corner."
+			"'s the pirates; a small number of them seem to be observing you."
+		word
+			" After some time of this "
+		word
+			"a silent alarm tells you that your should check on your ship. "
+			"you feel it may be wise to make sure no one's tampering with your ship. "
+		word
+			"You find "
+		word
+			"a lone pirate prodding the ship with rudimentary tools. "
+			"a few pirates watching but not apparently touching it. "
+			"no one there. The area around your ship is almost suspiciously devoid of people. "
+		word
+			"Whatever is going on, your ship appears unscathed."
+
+news "Raiders returning to Mordente-Bridi"
+	location
+		planet "Mordente-Bridi"
+	name
+		word
+			"Repairs"
+	message
+		word
+			"Watching the microgravity section of the spaceport from the bar, you "
+		word
+			"observe "
+			"notice "
+			"spot "
+			"spy "
+		word
+			"a Bastion "
+			"a Firebird "
+			"a Falcon "
+			"a Leviathan "
+		word
+			"returning from a raid, "
+		phrase
+			"Raiders returning to Mordente-Bridi - serious damage"
+			"Raiders returning to Mordente-Bridi - light damage"
+
+phrase "Raiders returning to Mordente-Bridi - serious damage"
+	word
+		"battered with thousands of small perforations. Somebody's turned a mainstay of the pirate arsenal, Gatling Guns, back on them"
+		"with substantial sections of hull heavily battered or missing altogether. It's a fair guess that they didn't have enough anti-missile"
+		"its hull covered in gashes and burns from Plasma Cannons"
+	word
+		". "
+	word
+		"As it comes to a stop in the maintenance bay, the Bridi engineering shuttles begin "
+	word
+		"buzzing "
+		"fluttering "
+	word
+		"around it, assessing the damage and beginning the work on repairs. "
+	word
+		"They'll be working for the rest of the shift on this one."
+
+phrase "Raiders returning to Mordente-Bridi - light damage"
+	word
+		"pockmarked with laser burns but with no apparent serious damage"
+		"lightly dented in a way that suggests that a Proton Gun was to blame"
+	word
+		". "
+	word
+		"As it comes to a stop in the maintenance bay, the Bridi engineering shuttles begin "
+	word
+		"buzzing "
+		"fluttering "
+	word
+		"around it, assessing the damage and beginning the work on repairs. "
+	word
+		"The work won't take the shuttles long."
+
+news "Symbol War"
```

Comment:
This news item also overflows. You'll need to shorten it somehow.
![image](https://github.com/user-attachments/assets/df8129d3-c582-48ea-a0d1-36b9d6cdc6bd)

Reply:
An alternative to shortening this would to have it as a culture conversation, rather than a news article.